spec/features/explore_user_profile_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'spec_helper'
+
+feature "User interacts with another users profile" do
+  given(:other_user) { Fabricate(:user) }
+  given(:other_user2) { Fabricate(:user) }
+  given!(:video1) { Fabricate(:video, reviews: [Fabricate(:review, user: other_user),
+                                                Fabricate(:review, user: other_user2)]) }
+  given!(:queued_video) { Fabricate(:queued_video, user: other_user, video: video1) }
+
+  scenario "by visiting another users profile" do
+    sign_in
+
+    #Go to user profile
```

Comment:
You are visiting a video here, not a profile. One of the biggest problems with comments is that they easily become stale (wrong, outdated). Rather than have these comments, you should extract the code under each comment into new methods that are named similarly to your comments. This will remove the need for the comments and cause your code to be self documenting.